### PR TITLE
feat: swap out default backend from pairing to blst

### PR DESCRIPTION
**Pull request by cryptonemo**
Requires https://github.com/filecoin-project/bellperson/pull/203

### CODE REVIEW

**Review by dignifiedquire — DISMISSED** [review]

**Comment by cryptonemo** [comment]
Resolves #1438

**Comment by cryptonemo** [comment]
Requires `bellperson 0.15.0` and a (breaking change) neptune release that also incorporates it.

**Comment by cryptonemo** [comment]
This PR will need a new review as it's getting ready since a number of deps are changing, including `neptune` usage, which has an updated API.

File: storage-proofs-porep/src/stacked/vanilla/proof.rs [hunk 1]
```diff
@@ -1422,11 +1432,13 @@ impl<'a, Tree: 'static + MerkleTreeTrait, G: 'static + Hasher> StackedDrg<'a, Tr
             let max_gpu_tree_batch_size = SETTINGS.max_gpu_tree_batch_size as usize;
 
             let _gpu_lock = GPU_LOCK.lock().expect("failed to get gpu lock");
+            let batcher = Some(
+                Batcher::pick_gpu(max_gpu_tree_batch_size).expect("failed to create gpu batcher"),
+            );
             let mut tree_builder = TreeBuilder::<Tree::Arity>::new(
                 #[cfg(feature = "gpu")]
-                Some(BatcherType::OpenCL),
+                batcher,
```
Comment (vmx) [inline storage-proofs-porep/src/stacked/vanilla/proof.rs:1440]: The config flag in the line above is not needed as the whole function is only enabled when the `gpu` feature is enabled.

Comment (cryptonemo) [reply storage-proofs-porep/src/stacked/vanilla/proof.rs:1440]: Agreed, I noticed it was there, so preserved it for now (i.e. left alone).  Can remove before merging.

**Review by DrPeterVanNostrand — APPROVED** [review]
Looks good to me

**Pull request merged**